### PR TITLE
Things to help develop&debug communication with remote server

### PR DESCRIPTION
**Pull request by dborzov**
This PR adds stuff that helps with developing and debugging of communication with the datacats remote server (datacats.io):
1) `scripts/test_remote_server.sh` to run the docker bash with bind mounts through which datacats communicates with a remote server
2) ssh_config fix to accept all hosts
3) Continue converting Exception Handling for `deploy` command to the one that gives informative error messages

@wardi plz CR

### CODE REVIEW

File: datacats/scripts/known_hosts [hunk 1]
```diff
@@ -1,3 +1,4 @@
 command.datacats.com ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDDyzfL3RJl1zDZoOKOh8gYGbqZ5wlDX7703leEpdl9242MnFf+29V0XNBWnZ/bRNwHS59NZUYjLg60TYscAkGW+Mp5bJ3O/csqDhYJ2Jy6kxjuiWn0t8whS//Wg0DgsW3v0pxB5A2i/QHP5FJdKdUvVlWkZAoEkoHhXwPgioGhNGx+/RU0AIMBOM2cn60aX0IXx5xtQHV9gCc88pwUbk+PwCIomJbnlrOscHLwWgj9xEGgxDgRDIHjiaBvVAiWJu9FSQXM4xz5ngsTyAz9iy6dL22yfm7yXwgFlOuNsVZL5qkunfJkRdkUDxWyTkvVtxIFHe0vo5FpAcYJV0KQeYDt
 command.datacats.com ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBGzB1OgWAWMxhYLV8raMJniU3Qdes6P3QYOhouNK53ChaaNU93vIZAbe3X5HJWLPrl1IbZPVvlq0nQB0P8EXcwg=
 
+192.168.0.13 ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDpH6rYXkR5bqKHDRdZN4enLJC0I3b039SH3rl4siJQlgRCffB1VOe1Sz1Bl6/f5kGglRXvLMV49uslUEh5ej0r0YY7Io6+2BOpA7a/SWLWFXM/eyKIVA917VK8zeHZczwf91bl0P3Vw4vmNWei4wvd6kBKR3wNOppEM4aAb6w17YjehR6F1xSagnbXsIcvrDSe73ym/7RJtcJM30rA8vH3WWsPpuZf1G2/TZAmi0ky44zDwAFBPj+nmUBIG8dhAW2UaLaKttD7VLbk7NKo6Okq6nnT4q++BOEmbq6qpspabfTJp7PZIURhx8ygvU5I4AtE0rF6B0qO2iEabY/VPiRB
```
Comment (JackMc) [inline datacats/scripts/known_hosts]: This being hardcoded is a security risk. A lot of networks use the 192.168.0.* namespace.

Comment (wardi) [inline datacats/scripts/known_hosts]: not a security risk (it's just a public key) but the key should be added a different way, maybe next to the deploy_target in the .datacats-environment file

Comment (JackMc) [inline datacats/scripts/known_hosts]: Yeah, sorry. My paranoia kicking in :smile:

File: datacats/scripts/test_remote_server.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# run bash within datacats/web with bind mounted ssh keys&config files
+# so that you can ssh to the remote server within the docker
+
+docker run \
+    -it \
+    -v  ~/.datacats/user-profile/id_rsa:/root/.ssh/id_rsa \
+    -v  ~/datacats/datacats/scripts/known_hosts:/root/.ssh/known_hosts \
+    -v  ~/datacats/datacats/scripts/ssh_config:/etc/ssh/ssh_config \
+        datacats/web \
+            bash
```
Comment (wardi) [inline datacats/scripts/test_remote_server.sh]: The files in the scripts dir are just things that we're bind mounting into containers so this doesn't fit here. Also, this seems more like a development helper that most people won't need, especially now that you've improved all the errors.

**Comment by wardi** [comment]
Remove the ssh host key and debug script then please merge

**Pull request merged**